lib/librarian/ansible/source/galaxy.rb

```diff
@@ -58,13 +58,13 @@ def lookup_package(username, name)
                 break if json['results'].nil?
 
                 package = json['results'].find do |r|
-                  r['namespace'] == username &&
-                    r['name'] == name
+                  r['summary_fields']['namespace']['name'] == username &&
+                      r['name'] == name
                 end
                 return package if package
 
                 break unless json['next']
-                url = "#{@@galaxy_api}/#{json['next']}"
+                url = "#{@@galaxy_api}#{json['next']}"
```

Comment:
This would be better expressed as: `File.join(@@galaxy_api, json['next'])`. That will automatically handle whether there’s a slash or not.